app/models/operatingsystems/redhat.rb

```diff
@@ -72,4 +76,21 @@ def pxe_kernel_options(params)
   def minor_version_help
     '0, 6.1810'
   end
+
+  def fedora?
+    name == 'Fedora'
+  end
+
+  def rhel_compatible?
```

Comment:
Where do we place Amazon Linux here? It's within the Red Hat ecosystem, but it's based on Fedora with its own independent release cycle.

Since you're including this in core, we're on the hook unlike if we kept the logic in the templates.